windows/launch.bat

```diff
@@ -0,0 +1,10 @@
+@ECHO OFF
+
+ECHO Launching Homebrewery...
+ECHO.
+ECHO =====================================================================
+ECHO.
+
```

Comment:
It's my understanding that MongoDB installs itself as a service, set to automatically start, so unless someone specifically messes with the configuration, it will be running.  
However, assumption is the mother of... mistakes, so it would be best to confirm prior to running `node server.js`.

`sc query mongodb | find "RUNNING"` or something to that effect.

Reply:
I intentionally never install it as a service; I found it to slow down my (admittedtly, old) development machine on startup and I just didn't need it going all the time, since I was also using it for work/school and sometimes it conflicts with those tasks.